src/Nokia_LCD.h

```diff
@@ -180,7 +200,8 @@ class Nokia_LCD {
      */
     bool printCharacter(const unsigned char character);
 
-    const uint8_t kClk_pin, kDin_pin, kDc_pin, kCe_pin, kRst_pin;
+    const uint8_t kClk_pin, kDin_pin, kDc_pin, kCe_pin, kRst_pin, kBl_pin;
     uint8_t mX_cursor, mY_cursor;
     bool inverted = false;
+    bool usingBacklight;
```

Comment:
Since `usingBacklight` is not going to change, let's turn it into a constant and name it appropriately (i.e. `kUsingBacklight`).

Reply:
Done!